test/presubmit-tests.sh

```diff
@@ -42,6 +44,22 @@ warn() {
   echo "WARN: $@"
 }
 
+install-node() {
+  info Installing node
+
+  curl -sL https://deb.nodesource.com/setup_14.x | bash -
+  apt-get install -y nodejs
+
+  npm install
+
```

Comment:
could you move `npm install` outside this function into `build`? 
Also, could you please run `node version` ?